src/common/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import IconButton from './IconButton'
+
+export default function Header() {
+  const history = useHistory()
+
+  return (
+    <HeaderContainer>
+      <IconButton
+        imgUrl="/img/back.svg"
+        altText="zurück"
+        onClick={() => history.goBack()}
+      />
+      <img src="/img/logo.svg" alt="Youlity logo" />
```

Comment:
With react router you could also import your image as react component.

Reply:
Yes, but I think if I import it as a component I can't set an alt text.